task.go

```diff
@@ -0,0 +1,10 @@
+package tasks
+
+// Структура Task описывает модель задачи с полями ID, Date, Title, Comment, и Repeat.
+type Tasks struct {

```

Comment:
Структура описывает одну задачу
```suggestion
type Task struct {

```